spec/support/contracts/json_schema_test/app/publish.mson

```diff
@@ -25,7 +25,7 @@ A post that only has a title, but no text body. Weird, right?
 - body: (string) - Body of the post
 - author_id: 2 (number, required) - User id of author
 - primary_tag: (App::Tag, optional) - Property of custom type
-- tags: (array[Tag]) - An array with a custom type
+- tags: (array[App::Tag]) - An array with a custom type
```

Comment:
It seems this is not possible anymore :/. The AST then contains no data structures, just an error saying: "Error code 4: base type 'Tag' is not defined in the document".